MdePkg/Include/Library/DebugLib.h

```diff
@@ -19,12 +19,14 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 //
 // Declare bits for PcdDebugPropertyMask
 //
-#define DEBUG_PROPERTY_DEBUG_ASSERT_ENABLED       0x01
-#define DEBUG_PROPERTY_DEBUG_PRINT_ENABLED        0x02
-#define DEBUG_PROPERTY_DEBUG_CODE_ENABLED         0x04
-#define DEBUG_PROPERTY_CLEAR_MEMORY_ENABLED       0x08
-#define DEBUG_PROPERTY_ASSERT_BREAKPOINT_ENABLED  0x10
-#define DEBUG_PROPERTY_ASSERT_DEADLOOP_ENABLED    0x20
+#define DEBUG_PROPERTY_DEBUG_ASSERT_ENABLED        0x01
+#define DEBUG_PROPERTY_DEBUG_PRINT_ENABLED         0x02
+#define DEBUG_PROPERTY_DEBUG_CODE_ENABLED          0x04
+#define DEBUG_PROPERTY_CLEAR_MEMORY_ENABLED        0x08
+#define DEBUG_PROPERTY_ASSERT_BREAKPOINT_ENABLED   0x10
+#define DEBUG_PROPERTY_ASSERT_DEADLOOP_ENABLED     0x20
+#define DEBUG_PROPERTY_ASSERT_BREAKASSERT_ENABLED  0x40 // MU_CHANGE BREAKASSERT

```

Comment:
I think the baselib and debuglib changes should be broken up into 2 commits to keep their own sanity?

Reply:
Split into 3 commits, an additional one for the new Pi Status codes. though re: your other comment, one of the new Pi codes does not seem related.